rocAL_pybind/CMakeLists.txt

```diff
@@ -19,9 +19,13 @@
 # THE SOFTWARE.
 
 cmake_minimum_required(VERSION 3.5)
-project(rocAL_pybind)
+set(VERSION "2.0.0")
 
-set(TARGET_NAME rocAL_pybind)
+# Set Project Version and Language
+project(rocal_pybind VERSION ${VERSION} LANGUAGES CXX)
+
+set(TARGET_NAME rocal_pybind)
```

Comment:
@kiritigowda Seems like you changed project and target from `rocAL_pybind` to 'rocal_pybind`. This might effect the build for python backend since the folder is still `rocAL_pybind`. Have you tested the python unit tests after this change? This target name is used in the wheel name etc